README.md

```diff
@@ -10,7 +10,7 @@
 <h1 align="center">Water.css</h1>
 <p align="center">🌊 A drop-in collection of CSS styles to make simple websites just a little nicer</p>
```

Comment:
We should also update the GitHub repository's description to match this tag line I think. :)

Reply:
Done!